staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6291,6 +6312,8 @@ type NamespaceList struct {
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.0
+// +k8s:prerelease-lifecycle-gen:deprecated=1.6
```

Comment:
revert this line, this type is also used for the binding subresource and is polluting runs with deprecation warnings on endpoints which are not deprecated

Reply:
xref https://github.com/kubernetes/kubernetes/issues/125312

Reply:
I created https://github.com/kubernetes/kubernetes/pull/125540